docs/send-data/opentelemetry-collector/install-collector/windows.md

```diff
@@ -30,6 +30,14 @@ You can install our OpenTelemetry Collector using either of the following method
 * [Install script](#install-script)
 * [Manual step-by-step installation](#manual-step-by-step-installation)
 
+:::info
+Whitelist the following repository URLs to allow a seamless installation:
+
```

Comment:
We use "allowlist" and "denylist" instead of "whitelist" and "blacklist". https://help.sumologic.com/docs/contributing/style-guide/#cultural-neutrality

```suggestion
Allowlist the following repository URLs to allow a seamless installation:
```